controllers/deliveryController.js

```
@@ -0,0 +1,155 @@
+        updateFields.actualArrival = new Date();
+      }
+    }
+
+    if (delayReason) {
+      updateFields.delayReason = delayReason;
+    }
+
+    // Push driver’s current location to the route (if provided)
+    const updateOps = {};
+
+    if (location && location.latitude && location.longitude) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)